cli-tests/tests/output-dir.test.ts

```diff
@@ -60,45 +69,52 @@ describe("Set of --output-dir tests", () => {
 
     it("Compiler warning/error is presented", () => {
       expect(result.output).toMatch(/No input sources specified\.\s*Error\(s\) found\. Compilation aborted/i);
+      tmpDirZkSolc.removeCallback();
     });
 
     it("solc exit code == zksolc exit code", () => {
+      const args = [`--bin`, `--output-dir`, `${tmpDirSolc.name}`]
       const solcResult = executeCommand(solcCommand, args);
       expect(solcResult.exitCode).toBe(result.exitCode);
+      tmpDirSolc.removeCallback();
     });
   });
 
   //id1813
   describe(`Run ${zksolcCommand} with --output-dir - specific symbols`, () => {
-    const args = [`${paths.pathToBasicSolContract}`, `--bin`, `--output-dir`, `${paths.pathToBadOutputDir}`];
+    const tmpDirZkSolc = createTmpDirectory(`File!and#$%-XXXXXX`);
+    const tmpDirSolc = createTmpDirectory(`File!and#$%-XXXXXX`);
+    const args = [`${paths.pathToBasicSolContract}`, `--bin`, `--output-dir`, `${tmpDirZkSolc.name}`];
     const result = executeCommand(zksolcCommand, args);
 
     it("Exit code = 0", () => {
       expect(result.exitCode).toBe(0);
     });
 
     it("Custom dir is created", () => {
-      expect(isDestinationExist(paths.pathToBadOutputDir)).toBe(true);
+      expect(isDestinationExist(tmpDirZkSolc.name)).toBe(true);
+      tmpDirZkSolc.removeCallback();
     });
 
     it("--output-dir output is presented", () => {
       expect(result.output).toMatch(/Compiler run successful/i);
     });
 
     it("solc exit code == zksolc exit code", () => {
-      expect(removeDirectory(paths.pathToBadOutputDir)).toBe(true);
+      const args = [`${paths.pathToBasicSolContract}`, `--bin`, `--output-dir`, `${tmpDirSolc.name}`];
       const solcResult = executeCommand(solcCommand, args);
       expect(solcResult.exitCode).toBe(result.exitCode);
-      expect(removeDirectory(paths.pathToBadOutputDir)).toBe(true);
+      tmpDirSolc.removeCallback();
     });
   });
 
   //id1812 - different behaviour on CI on Linux
   if (os.platform() !== 'linux') {
     describe(`Run ${zksolcCommand} with --output-dir - output-dir - wrong permissions`, () => {
-      createDirectory(paths.pathToReadOnlyOutputDir)
-      changeDirectoryPermissions(paths.pathToReadOnlyOutputDir, 'r');
-      const args = [`${paths.pathToBasicSolContract}`, `--bin`, `--output-dir`, `${paths.pathToReadOnlyOutputDir}`];
+      const tmpDirZkSolc = createTmpDirectory();
+      // const tmpDirSolc = createTmpDirectory();
```

Comment:
Please, remove comments if unnecessary.

Reply:
This stub should be uncommented after the fix in the zksolc output. The teset itself should contain this code.

Reply:
It's a good practice to add something like:
`// TODO: uncomment after CPR-xxxx is fixed`